vertical-pod-autoscaler/README.md

```diff
@@ -1,14 +1,46 @@
 # Vertical Pod Autoscaler
 
-# Intro
+### Intro
 
 Vertical Pod Autoscaler (VPA) frees the users from necessity of setting
 up-to-date resource requests for their containers in pods.
 When configured, it will set the requests automatically based on usage and
 thus allow proper scheduling onto nodes so that appropriate resource amount is
 available for each pod.
 
-# For users
+### Architecture
```

Comment:
There's already a little section called Architecture below, could you please merge them?

Reply:
Merged.